com.thomasokken.plus42.yml

```diff
@@ -0,0 +1,43 @@
+id: com.thomasokken.plus42
+command: plus42dec
+
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
```

Comment:
```suggestion
  - --share=ipc
  - --socket=fallback-x11
  - --socket=wayland
  - --device=dri
```

Reply:
Is this particular change a "strong" or a "soft" suggestion? I would prefer to keep the comments, since when I come back to this weeks or months down the line (or somebody else does), it helps when they're not very familiar with Flatpak.

Reply:
useless comment obscure things. It's like comment that 2+2=4

Reply:
Done